pkg/stages/steps_init.go

```diff
@@ -60,9 +60,66 @@ func GetInitrdStage(sys values.System, logger types.KairosLogger) ([]schema.Stag
 			}...)
 		}
 
-		// TODO: Add kairos-network and kairos-sysext here from packages directly
-		// In fact, it maybe be time to drop those packages and just use the dracut modules as we can add the exact modules we need
-		// based on the distro and version
+		// Add proper network and systemd-sysext if needed
+		// We default to systemd-networkd and sysext enbled and if its ubuntu <= 22.04 we need to use the plain network module and
+		// disable sysext as they are not supported in those versions
+		networkModule := "systemd-networkd"
+		sysextModule := true
+
+		if sys.Distro == values.Ubuntu {
+			constraint, _ := semver.NewConstraint("<=22.04")
+			ver, err := semver.NewVersion(sys.Version)
+			if err != nil {
+				logger.Logger.Error().Msgf("Failed to parse the version %s: %s", sys.Version, err)
+				return []schema.Stage{}, err
+			}
+			// If its <= 22.04 we need to use the plain network module and disable sysext
+			if constraint.Check(ver) {
+				logger.Logger.Debug().Str("distro", string(sys.Distro)).Str("version", sys.Version).Msg("Using the plain network module and disabling sysext")
+				networkModule = "network"
+				sysextModule = false
+			}
+		}
+
+		if sys.Distro == values.RockyLinux || sys.Distro == values.AlmaLinux || sys.Distro == values.RedHat {
+			// On Rocky and AlmaLinux we need to use the plain network modulegit status
```

Comment:
```suggestion
			// On Rocky and AlmaLinux we need to use the plain network module
```

Reply:
So that's why my gut status command didn't work lol